bot/exts/moderation/watchchannels/talentpool.py

```diff
@@ -120,9 +121,7 @@ async def watch_command(self, ctx: Context, user: FetchedMember, *, reason: str
         )
 
         if history:
-            total = f"({len(history)} previous nominations in total)"
-            start_reason = f"Watched: {textwrap.shorten(history[0]['reason'], width=500, placeholder='...')}"
-            msg += f"\n\nUser's previous watch reasons {total}:```{start_reason}```"
+            msg += f"\n\n{len(history)} previous nominations in total"
```

Comment:
This should be in parentheses
```suggestion
            msg += f"\n\n({len(history)} previous nominations in total)"
```